graphql-activerecord.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "activesupport", ">= 4.2", '< 6'
   spec.add_runtime_dependency "activerecord", ">= 4.2", '< 6'
-  spec.add_runtime_dependency "graphql", ">= 1.5.10", '< 2'
+  spec.add_runtime_dependency "graphql", ">= 1.7.5", '< 2'
```

Comment:
@theorygeek, just curiosity, is this what causes the change from `type Foo implements Bar, Baz { field: Type }` to `type Foo implements Bar & Baz { field: Type }` in the resulting schema?

Reply:
probably....